app/src/main/java/com/keylesspalace/tusky/components/notifications/NotificationHelper.java

```diff
@@ -456,7 +456,7 @@ private static PendingIntent getStatusComposeIntent(Context context, Notificatio
 
         Intent composeIntent = MainActivity.composeIntent(context, composeOptions, account.getId(), body.getId(), (int)account.getId());
 
-        composeIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+        composeIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
```

Comment:
I'd leave a comment in code why we do this

Reply:
➕